### PR TITLE
PWGHF: add new channel in derived data production

### PR DESCRIPTION
**Pull request by ZFederica**
Add new channel in derived data production (omegac -> omega K) and remove xi track duplicate

### CODE REVIEW

**Review by fgrosa — CHANGES_REQUESTED** [review]
Hi @ZFederica thanks a lot for the implementation!
Generally it looks good to me, but if possible I would avoid to compute twice the Omega+track vertex (see my comment below).

File: PWGHF/TableProducer/trackIndexSkimCreator.cxx [hunk 1]
```diff
@@ -3638,53 +3645,94 @@ struct HfTrackIndexSkimCreatorLfCascades {
...
               }
             } else if (df2.isPropagationFailure()) {
-              LOGF(info, "Exception caught: failed to propagate tracks (2prong - omega) to charm baryon decay vtx");
+              LOGF(info, "Exception caught: failed to propagate tracks (2prong - omega pi) to charm baryon decay vtx");
+            }
+          }
+
+          // find charm baryon decay using omega PID hypothesis to be combined with a kaon
+          int nVtxFrom2ProngFitterOmegaHypWithK = 0;
+          try {
```
[the remark this answers is not in the document]
Comment (ZFederica) [reply PWGHF/TableProducer/trackIndexSkimCreator.cxx:3656]: Hi @fgrosa, the reason for the current implementation is that I should use two different trackParCov with different PID hyp for the charm bachelor track to be processed by the DCAFitter. I had overlooked this and forgot to set properly the PID, now it should be fine.

[the remark this answers is not in the document]
Comment (fgrosa) [reply PWGHF/TableProducer/trackIndexSkimCreator.cxx:3656]: Hi @ZFederica, is this really necessary? I understand for the cascades, but the bachelor track is a primary track already propagated to the PV by the track-propagation task, so the mass hypothesis will not have any effect for it. For instance we do not set any mass hypotheses for the charm hadrons with decay daughters that do not have V0s or cascades in the decay.

[the remark this answers is not in the document]
Comment (ZFederica) [reply PWGHF/TableProducer/trackIndexSkimCreator.cxx:3656]: Hi @fgrosa, I see, I will implement your first suggestion

**Review by fgrosa — APPROVED** [review]

**Pull request merged**